frontend/src/pages/website/Establishment/index.jsx

```diff
@@ -77,6 +78,21 @@ function Establishment() {
             />
           ))}
         </div>
+
+        <div className={styles.EstablishmentLeftServicesSection}>
+          <h2 className={styles.EstablishmentTitle}>
+            {t('establishementLocation')}
+          </h2>
+        </div>
+        <Map position={[ 48.8665, 2.3335 ]} markers={[
+          {
+            position: [ 48.8665, 2.3335 ],
+            popup: 'Popup 1',
+          },
+        ]}
+        zoomLevel={13}
+        />
```

Comment:
```suggestion
        <div className={styles.EstablishmentLeftServicesSection}>
          <h3 className={styles.EstablishmentSubtitle}>
            {t('establishementLocation')}
          </h3>
          
          <span className={styles.EstablishmentAddress}>
            {establishment?.street}, {establishment?.zipCode} {establishment?.city}
          </span>
          <Map position={[ 48.8665, 2.3335 ]} markers={[
            {
              position: [ 48.8665, 2.3335 ],
              popup: 'Popup 1',
            },
          ]}
          zoomLevel={13}
          />
        </div>
```

Met la map dans la section, utilise un subtitle en h3 au lieux d'un title h2, et affiche l'adress comme sur planity